src/box/txn_limbo.c

```diff
@@ -51,6 +51,7 @@ txn_limbo_create(struct txn_limbo *limbo)
 	limbo->confirmed_lsn = 0;
 	limbo->rollback_count = 0;
 	limbo->is_in_rollback = false;
+	limbo->freezed = false;
```

Comment:
Sorry for nitpicking, but it's `frozen`.

Reply:
Wow, thanks for noticing! Fixed